hadoop-ozone/dist/src/main/smoketest/balancer/testBalancer.robot

```diff
@@ -63,6 +63,22 @@ Datanode Recommission is Finished
 Run Container Balancer
     ${result} =             Execute                         ozone admin containerbalancer start -t 1 -d 100 -i 1
                             Should Contain                  ${result}             Container Balancer started successfully.
+    ${verboseStatusResult} =      Execute                         ozone admin containerbalancer status -v
+                            Should Contain                  ${verboseStatusResult}             ContainerBalancer is Running.
+                            Should Contain                  ${verboseStatusResult}             Started at:
+                            Should Contain                  ${verboseStatusResult}             Container Balancer Configuration values:
+                            Should Contain                  ${verboseStatusResult}             Current iteration info:
+                            Should Contain                  ${verboseStatusResult}             Iteration number                                   1
+                            Should Contain                  ${verboseStatusResult}             Iteration result                                   IN_PROGRESS
+                            Should Contain                  ${verboseStatusResult}             Scheduled to move containers                       3
+    ${statusHistoryResult} =    Execute                         ozone admin containerbalancer status -v -h
+                            Should Contain                  ${statusHistoryResult}             ContainerBalancer is Running.
+                            Should Contain                  ${statusHistoryResult}             Started at:
+                            Should Contain                  ${statusHistoryResult}             Container Balancer Configuration values:
+                            Should Contain                  ${statusHistoryResult}             Iteration history list:
+                            Should Contain                  ${statusHistoryResult}             Iteration number                                   1
+                            Should Contain                  ${statusHistoryResult}             Iteration result                                   IN_PROGRESS
+                            Should Contain                  ${statusHistoryResult}             Scheduled to move containers                       3
```

Comment:
> Please create a reusable keyword to check expected output.

Sorry, if my comment could be misinterpreted.

By "reusable keyword" I meant something like this:

```robot
Verify Balancer Status
    [arguments]    ${output}

    Should Contain    ${output}    ContainerBalancer is Running.
    Should Contain    ${output}    Started at:
    Should Contain    ${output}    Container Balancer Configuration values:

Verify Balancer Iteration
    [arguments]    ${output}    ${number}    ${status}    ${containers}
    Should Contain    ${output}    Iteration number                                   ${number}
    Should Contain    ${output}    Iteration result                                   ${status}
    Should Contain    ${output}    Scheduled to move containers                       ${containers}
```

Then the main test could be shortened:

```robot
    ${result} =    Execute    ozone admin containerbalancer status -v
    Verify Balancer Status Output    ${result}
    Verify Balancer Iteration        ${result}    1    IN_PROGRESS    3
    Should Contain                   ${result}    Current iteration info:

    ${result} =    Execute    ozone admin containerbalancer status -v --history
    Verify Balancer Status Output    ${result}
    Verify Balancer Iteration        ${result}    1    IN_PROGRESS    3
    Should Contain                   ${result}    Iteration history list:
```

We could add a keyword for checking output with history and another for the plain verbose output.  They could verify that one the right one of "Current iteration info" and "Iteration history list" is present, the other one is not.

Please feel free to change this example as needed.

Reply:
Extracted to separate test. Hope i properly understood )

Reply:
Thanks @juncevich for making the robot test changes.

I see 3 problems with the new `testBalancerStatus.robot`:

1. it is not run at all (see details below)
2. it duplicates code from `testBalancer.robot`
3. volume, bucket and key names will conflict with those created by `testBalancer.robot`, so the test will fail when trying to create those

We can fix all these problems by merging the new script back into `testBalancer.robot`, removing duplicated parts, and running test preparation only once.

----

Existing `testBalancer.robot` is run here:

https://github.com/apache/ozone/blob/e01a57d8988d89a5c8a985989ee5ac240decac5d/hadoop-ozone/dist/src/main/compose/ozone-balancer/test.sh#L33

```
Verify Container Balancer for RATIS containers                        | PASS |
------------------------------------------------------------------------------
testBalancer :: Smoketest ozone cluster startup                       | PASS |
1 test, 1 passed, 0 failed
```

https://github.com/juncevich/ozone/actions/runs/9972072803/job/27560052180#step:5:131

Reply:
@adoroszlai, thank you for infrormation. Will fix it.

Reply:
@adoroszlai, i think i'be fixed it )